### ISSUE TITLE
Directly output boolen mask from `transect_mask`?

### ISSUE THREAD

**Issue by leewujung** [OSOceanAcoustics/echoregions#208]
Right now the `transect_mask` function produces masks with 0/1 integer values, which can be turned into True/False boolean masks for downstream combination with other masks.

Wouldn't it be more straightforward to directly produce masks with True/False boolean values from this function?

**Comment by ctuguinay** [comment]
Yeah I think True/False would be for the best. And you're right, I think I had originally done this to match the existing mask functions, but I'll change that behavior in another refactoring.

**Comment by ctuguinay** [comment]
Closed by #298
